contributing/code/standards.rst

```diff
@@ -143,8 +143,9 @@ Structure
 * Add a blank line before ``return`` statements, unless the return is alone
   inside a statement-group (like an ``if`` statement);
 
-* Use just ``return;`` instead of ``return null;`` when a function must return
-  void early;
+* Use ``return null;`` when a function explicitly returns ``null`` values and
+  use ``return;`` when the function returns ``void`` values (available from
+  PHP 7.1);
```

Comment:
I am not sure if the last part wouldn't be confusing. Could people think that this isn't usable yet as their code needed to be compatible with older PHP versions too?